src/key_io.cpp

```diff
@@ -114,7 +116,7 @@ class DestinationEncoder : public boost::static_visitor<std::string>
             bytes.insert(bytes.end(), id.program, id.program + id.length);
             ConvertBits<8, 5, true>([&](unsigned char c) { data.push_back(c); }, bytes.begin(), bytes.end());
             const std::string& hrp = for_parent ? m_params.ParentBlech32HRP() : m_params.Blech32HRP();
-            return blech32::Encode(hrp, data);
+            return blech32::Encode(blech32::Encoding::BLECH32, hrp, data);
```

Comment:
Should this be BLECH32M? Compare to 5 lines down.

Reply:
Yep, good catch, thanks